pylint/interfaces.py

```diff
@@ -40,7 +41,7 @@ def is_implemented_by(cls, instance):
         return implements(instance, cls)
 
 
-def implements(obj, interface):
+def implements(obj: Any, interface: Tuple[type, type]) -> bool:
```

Comment:
```suggestion
def implements(obj: "Interface", interface: Tuple[type, type]) -> bool:
```
Although I think it might be possible to restrict this even more